.travis.yml

```diff
@@ -71,24 +78,30 @@ matrix:
        - NPY_BLAS_ORDER=mkl,blis,openblas,atlas,accelerate,blas
        - NPY_LAPACK_ORDER=MKL,OPENBLAS,ATLAS,ACCELERATE,LAPACK
```

Comment:
Anyone know why these environmental variables were added? I don't think we use any of those.